src/components/cluster/Cluster.tsx

```diff
@@ -514,7 +514,7 @@ function ClusterForm({
 
     async function onValidation() {
         let payload = getClusterPayload()
-
+        state.url.value = state.url.value.trim()
```

Comment:
Can we have a new var instead of re-assigning the trimmed value?

Reply:
Done